server/py/framework/db/sqldb/db.py

```diff
@@ -60,11 +60,16 @@
 import mlrun.model
 import mlrun.utils.db
 from mlrun.artifacts.base import fill_artifact_object_hash
+from mlrun.common.schemas import FunctionState
```

Comment:
No need to import `FunctionState`, we are already importing `mlrun.common.schemas` so you can use `mlrun.common.schemas.FunctionState`
```suggestion

```